config/README.md

```diff
@@ -62,6 +63,14 @@ postgres://user:pass@db/utask?sslmode=disable
             "config": {
                 "webhook_url": "https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX"
             }
+        },
+        "webhook-example.org": {
```

Comment:
is the payload format documented somewhere? I dont see it

Reply:
done in README.md